packages/treasury/src/stablecoinMachine.js

```diff
@@ -284,6 +285,21 @@ export async function start(zcf) {
     return rewardPoolSeat.getCurrentAllocation();
   }
 
+  function getBootstrapPayment() {
```

Comment:
This function allows calling it as many times as you want, which seems incorrect.  Can you make it self-destruct after the first minted payment?

Reply:
ah, very good point. I'll do that

Reply:
Fixed